2-import.R

```diff
@@ -0,0 +1,320 @@
+##############################################
+#               Functions                    #
+##############################################
+
+
+########################
+# Retrieve the data associated to a sensor
+########################
+
+
+#'  Retrieve the data associated to a sensor from the API Telraam
+#'
+#' @param nom Character. Name of the sensor, comes from the vector listeNom
```

Comment:
Is it possible to rename variables into English?
This suggestion applies for all files.